src/content/structured/components/popover-menu/accessibility.mdx

```diff
@@ -0,0 +1,51 @@
+---
+path: "/components/popover-menu/accessibility"
+
+date: "2023-03-31"
+
+title: "Popover menu"
+
+status: "BETA"
+
+subTitle: "Use popover menus to place actions inside a menu that overlays other page content when opened."
+
+contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/structured/components/popover-menu"
+
+tabs:
+  [
+    { title: "Guidance", path: "/components/popover-menu" },
+    { title: "Code", path: "/components/popover-menu/code" },
+    { title: "Accessibility", path: "/components/popover-menu/accessibility" },
+  ]
+---
+
+## Easy to use for everyone
+
+Popover menus use the role ‘menu’ and the items within use ‘menuitem’ and ‘menuitemcheckbox’. This enables the menu to be opened using usual keyboard commands. Clicking on a menu item closes the menu after triggering the menu item's action. A menu item's action can also be triggered using the "Space" and "Enter" keys. Popover menu groups use the role ‘group’.
+
+Popover menus appear overlaid on other content. When opening a submenu, focus is applied to the whole sub-menu and then the items are navigated between using the up and down arrow keys. When focus is moved on, the popover menu closes automatically. Using the ‘Esc’ key will also close the popover menu.
+
+Disabled menu items can still be focussed upon so that the structure of the menu is understandable, even if the menu item can’t be triggered at that time.
+
+## For Assistive Technology
+
+Each menu item within a popover menu can use an aria-label that provides additional context that may not be required in a visible label.
+
+For ease of use, actions can be assigned keyboard shortcuts and these are displayed on the menu item. This information is announced by a screen reader when each menu item is focussed.
```

Comment:
I'm not getting this behaviour when looking at 'show found results'

Reply:
I think this is because it closes to complete an action before it has the chance to read it out. @GCHQ-Developer-112 do you know if this is intended behaviour or if the popover needs updating to not close the menu when toggle menu buttons are clicked in order to read out the state?

Reply:
I've removed the line in the accessibility guidance as aria-pressed isn't correct as that is only for role=button